### ISSUE TITLE
Exploding loss in training

### ISSUE THREAD

**Issue by Mayamayb** [ucaswangls/EfficientSCI#3]
Hello,
Thanks for sharing your code.
I cloned the repo and began training on the Davis 480p dataset with the default configs from the repo.
After ~93K iterations, during the 3rd epoch, the loss suddenly began to go up very quickly and exploded, failing the training.
Has this happened to you? can you please share the training details?
Best regards.
![image](https://github.com/ucaswangls/EfficientSCI/assets/37766793/1b657509-3818-4c92-b7a2-d24b29e27a95)

**Comment by geekOctopus** [comment]
I meet the same problem today, maybe you can try to set the learning rate to 0.00001
